src/UNL/UCBCN/ActiveRecord/Record.php

```diff
@@ -283,13 +283,12 @@ public static function __callStatic($method, $args)
             }
 
             return self::getByAnyField($class, $field, $args[0], $whereAdd);
-
+            break;
```

Comment:
Technically, no, but it is a good practice anyway. If the return is later removed, the user may not realize a break needs to be inserted.